app/javascript/components/profile/relations.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div class="profile-relations">
+        <div class="profile-relations__title">
+            {{ $i18n.t('message.profile.relationsTitle') }}
+        </div>
+        <div class="profile-relations-users">
+          <div v-if="beingFetched || !recommendations" class="loading-icon"></div>
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'class' should be on a new line                                         vue/max-attributes-per-line